crates/runtime/tests/models/search.rs

```diff
@@ -554,15 +554,24 @@ async fn test_text_search_multiple_columns() -> Result<(), anyhow::Error> {
             SearchTestCase {
                 name: "multi_text_column_basic",
                 body: json!({
-                    "text": "general",
+                    "text": "In general basic",
```

Comment:
More specific query so that identical scored results do not break consistency of testing.